spec/manuals_publisher/removing_content_spec.rb

```diff
@@ -0,0 +1,59 @@
+feature "Removing content on Manuals Publisher", manuals_publisher: true do
+  include ManualsHelpers
```

Comment:
Just noticed that we seem to have broken a consistency with the naming here, other helpers having publisher in name. e.g.`SpecialistPublisherHelpers`